pydicom/valuerep.py

```diff
@@ -415,17 +456,15 @@ def __init__(self, val):
         # a different Decimal, as Decimal is immutable.
         if isinstance(val, (str, compat.text_type)):
             self.original_string = val
-        elif isinstance(val, (DSfloat, DSdecimal)) and hasattr(val, 'original_string'):
+        elif isinstance(val, (DSfloat, DSdecimal)) and hasattr(val, 'original_string'):  # noqa
             self.original_string = val.original_string
 
     def __str__(self):
-        if hasattr(self, 'original_string') and len(self.original_string) <= 16:
+        if hasattr(self, 'original_string') and len(self.original_string) <= 16:  # noqa
             return self.original_string
         else:
             return super(DSdecimal, self).__str__()
 
-    def __repr__(self):
-        return "\"" + str(self) + "\""
 
```

Comment:
A mistake by me - I had tried to find a solution for the above that would work (and wound up using noqa and returning to the original) and when I reverted back I missed the last bit that used to be there.